src/test/java/reposense/util/TestConstants.java

```diff
@@ -5,12 +5,7 @@
 public class TestConstants {
     public static final String TEST_ORG = "reposense";
     public static final String TEST_REPO = "testrepo-Alpha";
+    public static final String TEST_MASTER_BRANCH = "master";
```

Comment:
can we shift this to git checkout? also, make a default method that checkouts to master?

Reply:
Actually, I think we can should use `config.getBranch()` instead, so no need for this 'constant'. Will remove it.